homeassistant/components/recswitch.py

```diff
@@ -0,0 +1,39 @@
+"""
+Support for Ankuoo RecSwitch MS6126 devices.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/switch.recswitch/
```

Comment:
The url is wrong.

Reply:
I'll write the documentation.. I promise!